packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -36,7 +36,7 @@ export const startPSM = async (
       consume: { contractGovernor, psm: psmInstallP },
     },
     instance: {
-      consume: { economicCommittee },
+      consume: { economicCommittee: economicCommitteeP },
```

Comment:
the readme calls out `startPSM` as exemplary, to be emulated. I think this new deeplyFulfilled technique is the state of the art so please consider updating `startPSM` to use it. If not maybe I will later. cc @dckc

Reply:
done.